account_manager/src/keystore/checksum.rs

```diff
@@ -0,0 +1,28 @@
+use crypto::digest::Digest;
+use crypto::sha2::Sha256;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+
+/// Checksum module for `Keystore`.
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+pub struct ChecksumModule {
+    pub function: String,
+    pub params: BTreeMap<(), ()>, // TODO: need a better way to encode empty json object
```

Comment:
Perhaps [`serde_json::Value`](https://github.com/serde-rs/json#operating-on-untyped-json-values) would work?

Reply:
Yup that's perfect. Thanks! :)